.github/workflows/main.yml

```diff
@@ -25,6 +27,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Free Disk Space
+        uses: ./.github/workflows/free-disk-space
       - name: Build image
         run: ./kas-container build kas-iot2050-example.yml:kas/opt/preempt-rt.yml
       - name: Upload image
```

Comment:
In https://github.com/siemens/meta-iot2050/pull/259/commits/e3ec3055d7b1c3d40eecfc0f02de62fea7dedd55, I'm applying it to the firmware build as well, just to be consistent.